src/formatDate.js

```diff
@@ -50,7 +50,31 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateStr = date.split(fromFormat[3]);
+  const fullYear = 'YYYY';
+  const shortYear = 'YY';
+  const dateObj = {};
+  const newDate = [];
+
+  for (let i = 0; i < 3; i++) {
```

Comment:
Again magic?

Reply:
Ok, but you also can use array length here instead of 3